spot_rl_experiments/experiments/skill_test/test_nav_pick_nav_place_skills.py

```diff
@@ -19,7 +19,9 @@
             enable_force_control=True,
         )
         spotskillmanager.nav("kitchen_counter")
-        spotskillmanager.place(None, is_local=True, visualize=False)
+        spotskillmanager.place(
+            "", is_local=True, visualize=False, enable_waypoint_estimation=True
```

Comment:
we can still use spotskillmanager.place(None, ...) right? if so, let's change "" back to None